sonar-batch/src/main/java/org/sonar/batch/index/DefaultIndex.java

```diff
@@ -221,6 +203,10 @@ public Measure getMeasure(Resource resource, org.sonar.api.batch.measure.Metric<
   public Measure addMeasure(Resource resource, Measure measure) {
```

Comment:
:new: ![MAJOR](https://raw.githubusercontent.com/henryju/image-hosting/master/major.png)The Cyclomatic Complexity of this method "addMeasure" is 11 which is greater than 10 authorized.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)